rustfmt.toml

```diff
@@ -0,0 +1,55 @@
+max_width = 90
```

Comment:
Are these standard configurations? what are the customizations we are making?

Reply:
No, I had to change some of them. Changes are below (`<` - our file, `>` - default style):

```
1c1
< max_width = 90
---
> max_width = 100
```

```
3c3
< tab_spaces = 2
---
> tab_spaces = 4
```

```
8,9c8,9
< wrap_comments = true
< comment_width = 90
---
> wrap_comments = false
> comment_width = 80
```

```
19,20c19,20
< reorder_imports = true
< reorder_imports_in_group = true
---
> reorder_imports = false
> reorder_imports_in_group = false
```

```
49c49
< # required_version = "0.3.8"
---
> required_version = "0.3.8"
```

```
55c55
< error_on_unformatted = true
---
> error_on_unformatted = false
```